csharp/src/Apache.Arrow.Adbc/Results.cs

```diff
@@ -43,7 +43,7 @@ public QueryResult(long rowCount, IArrowArrayStream stream)
         /// <summary>
         /// The number of records in the result.
         /// </summary>
-        public long RowCount { get; set; }
+        public long RowCount { get; }
```

Comment:
will need a variable to set the rowCount to in the constructor

Reply:
This is how you declare a readonly auto property. You can set it in the ctor but nowhere else.